src/sentry/utils/pytest/selenium.py

```diff
@@ -78,6 +79,63 @@ def set_emulated_media(self, features, media=""):
             "Emulation.setEmulatedMedia", {"media": media, "features": features}
         )
 
+    def set_window_size(self, width=None, height=None, remember_previous_size=False):
+        """
+        Sets the window size.
+
+        If width is not passed, then use current window width (this is useful if you
+        need to fit contents height into the viewport).
+        If height is not passed, then resize to fit the document contents.
+
+        Optionally, remember the previous window size so that it can later be restored.
+        This will never overwrite an existing saved size, so make sure you clean up with
+        `reset_window_size()`
```

Comment:
Should this method be a context manager so that you can't forget to resize the window?